### PR TITLE
feat: tags assign to pgd

### PR DESCRIPTION
**Pull request by wai-wong-edb**

### CODE REVIEW

File: pkg/provider/resource_cluster.go [hunk 1]
```diff
@@ -1226,3 +1221,26 @@ func StringSliceToSet(items *[]string) types.Set {
 
 	return types.SetValueMust(types.StringType, eles)
 }
+
```
Comment (edbamolvyavahare) [inline pkg/provider/resource_cluster.go:1224]: add a one-liner comment to describe purpose of the function.

File: pkg/provider/resource_cluster.go [hunk 2]
```diff
@@ -1226,3 +1221,26 @@ func StringSliceToSet(items *[]string) types.Set {
...
+	for _, v := range *apiRespTags {
+		*tfRsrcTags = append(*tfRsrcTags, commonTerraform.Tag{
+			TagId:   types.StringValue(v.TagId),
+			TagName: types.StringValue(v.TagName),
+			Color:   basetypes.NewStringPointerValue(v.Color),
+		})
+	}
+}
+
+func buildAPIReqAssignTags(tfRsrcTags []commonTerraform.Tag) []commonApi.Tag {
```
[the remark this answers is not in the document]
Comment (wai-wong-edb) [reply pkg/provider/resource_cluster.go:1236]: I moved this function with a comment into common in a later PR https://github.com/EnterpriseDB/terraform-provider-biganimal/pull/533

File: pkg/provider/resource_cluster.go (hunk 1 above)
Comment (wai-wong-edb) [reply pkg/provider/resource_cluster.go:1224]: I moved this function with a comment into common.go in a later PR https://github.com/EnterpriseDB/terraform-provider-biganimal/pull/533

**Review by edbamolvyavahare — APPROVED** [review]
LGTM

**Pull request merged**